docker-compose/monitor/spm-v2.yml

```diff
@@ -0,0 +1,38 @@
+service:
```

Comment:
you are missing the actual span metrics connector

Reply:
Yes, I wanted to test the current config with the local build first then I thought of adding the configuration for the span metrics connector.